lib/src/widget.dart

```diff
@@ -61,7 +62,7 @@ class AdvancedCalendar extends StatefulWidget {
   final int? startWeekDay;
 
   /// Style of headers date
-  final TextStyle? dateStyle;
+  final TextStyle? headerStyle;
```

Comment:
please add docs to the readme file regarding to the new props:
- headerStyle
- calendarTextStyle

Reply:
Updated